### DISCUSSION TITLE
auto sit in bat fle it's possible?

### DISCUSSION THREAD

**Discussion by katie0810ts** [cinderblocks/radegast#54] [General]
Hi everyone, I wanted to know if it is possible to also insert the autosit parameter in the bat file, thanks

I show you an example of my bat if after land I can insert autosit and then with uuid obviously

START ""/D "C:\Program Files\Radegast\" "C:\Program Files\Radegast\Radegast.exe" -a -g agni -u "ava" -p "avaps" -l "land"

**Comment by angelicdollbaby** [comment]
The radegast preferences have a setting where you can put an UUID in for an object and it will make the avi sit on it.
https://gyazo.com/cdd229af2bb91ac446f63808bd3ce829
Open preferences and click the automation tab. It is my favorite thing in radegast. Now if it would stop breaking my doll that would be great.

**Comment by katie0810ts** [comment]
yes thanks I knew this, but I needed it in a bat list with 20 ava to make them sit in the exact spot and each one different
